DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/lacp/test_lacp_loopback_detection.py

```diff
@@ -123,6 +125,9 @@ async def test_lacp_loopback_detection(testbed, version):
     await tgen_utils_stop_traffic(tgen_dev)
 
     # 7. Verify device remain stable afterwards (during storming).
+    out = await Ethtool.show(input_data=[{dent: [{'devname': dut_ixia_ports[2]}]}],  parse_output=True)
+    speed = int(out[0][dent]['parsed_output']['speed'][:-4])
+    expected_rate = round(int(expected_rate * speed), 2)
```

Comment:
Please move this to step 1 or step 6, so that you know the line rate of the port before you start storming the device (and potentially getting stuck here).